fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowColumnsCommand.java

```diff
@@ -0,0 +1,256 @@
+            List<List<String>> rows = Utils.executePlan(ctx, executor, plan);
+            for (List<String> row : rows) {
+                String rawType = row.get(1);
+                row.set(1, normalizeSqlColumnType(rawType));
+            }
+
+            return new ShowResultSet(metaData, rows);
+        }
+        List<List<String>> rows = Lists.newArrayList();
+        String ctl = tableNameInfo.getCtl();
+        DatabaseIf db = Env.getCurrentEnv().getCatalogMgr().getCatalogOrAnalysisException(ctl)
+                .getDbOrAnalysisException(tableNameInfo.getDb());
+        TableIf table = db.getTableOrAnalysisException(tableNameInfo.getTbl());
+        PatternMatcher matcher = null;
+        if (likePattern != null) {
+            matcher = PatternMatcherWrapper.createMysqlPattern(likePattern,
+                    CaseSensibility.COLUMN.getCaseSensibility());
+        }
+        table.readLock();
+        try {
+            List<Column> columns = table.getBaseSchema();
+            for (Column col : columns) {
+                if (matcher != null && !matcher.match(col.getName())) {
+                    continue;
+                }
+                final String columnName = col.getName();
+                final String columnType = col.getOriginType().toString().toLowerCase(Locale.ROOT);
+                final String isAllowNull = col.isAllowNull() ? "YES" : "NO";
+                final String isKey = col.isKey() ? "YES" : "NO";
+                final String defaultValue = col.getDefaultValue();
+                final String aggType = col.getAggregationType() == null ? "" : col.getAggregationType().toSql();
+                if (isFull) {
+                    // Field Type Collation Null Key Default Extra
+                    // Privileges Comment
+                    rows.add(Lists.newArrayList(columnName,
+                            columnType,
+                            "",
+                            isAllowNull,
+                            isKey,
+                            defaultValue,
+                            aggType,
+                            "",
+                            col.getComment()));
+                } else {
+                    // Field Type Null Key Default Extra
+                    rows.add(Lists.newArrayList(columnName,
+                            columnType,
+                            isAllowNull,
+                            isKey,
+                            defaultValue,
+                            aggType));
+                }
+            }
+        } finally {
+            table.readUnlock();
+        }
+
+        return new ShowResultSet(metaData, rows);
+    }
+
+    @Override
+    public <R, C> R accept(PlanVisitor<R, C> visitor, C context) {
+        return visitor.visitShowColumnsCommand(this, context);
+    }
+
+    @Override
+    public ShowResultSetMetaData getMetaData() {
+        return metaData;
```

Comment:
getMetaData() may be called by prepare statement, we need return correct info like bellow
```
if (isFull) {
    return META_DATA_VERBOSE;
} else {
   return META_DATA;
}
```